pkg/skaffold/debug/transform_python.go

```diff
@@ -0,0 +1,188 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package debug
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+)
+
+type pythonTransformer struct{}
+
+func init() {
+	containerTransforms = append(containerTransforms, pythonTransformer{})
+}
+
+const (
+	// most examples use 5678
+	defaultPtvsdPort = 5678
+)
+
+// ptvsdSpec captures the useful nodejs devtools options
+type ptvsdSpec struct {
+	host string
+	port int32
+	wait bool
+}
+
+// isLaunchingPython determines if the arguments seems to be invoking python
+func isLaunchingPython(args []string) bool {
+	return args[0] == "python" || strings.HasSuffix(args[0], "/python") ||
+		args[0] == "python2" || strings.HasSuffix(args[0], "/python2") ||
+		args[0] == "python3" || strings.HasSuffix(args[0], "/python3")
+}
+
+func (t pythonTransformer) IsApplicable(config imageConfiguration) bool {
+	if _, found := config.env["PYTHON_VERSION"]; found {
```

Comment:
is this `env["PYTHON_VERSION"]` on the container config? Is it possible that some java(or other language) application shells out to some python application to gets something done (like gcloud for instance) and would include this `env` value?

Reply:
It is on the container configuration for [the standard images](https://github.com/GoogleContainerTools/distroless/issues/276#issuecomment-460375014)